src/Template/Element/Modules/index_table_row.twig

```diff
@@ -26,7 +26,7 @@
                 </div>
             {% else %}
                 <div class="{{ prop }}-cell" untitled-label="{{ __('Untitled') }}">
-                    {% set propVal = object.attributes[prop] ?? object.meta[prop] %}
+                    {% set propVal = Property.objectValue(object, prop) %}
```

Comment:
we could probably use `Schema.format` inside `Property` method call & introduce a `bytes` primitive type to format `file_size` - more details later also on this subject